lib/databaseOps.js

```diff
@@ -0,0 +1,39 @@
+var mysql = require("mysql2");
+
+module.exports = {
+  connectToDB: () => {
+    var con = mysql.createConnection({
+      host: process.env.MYSQL_HOSTNAME,
+      user: process.env.MYSQL_USER,
+      password: process.env.MYSQL_PASSWORD,
+      database: process.env.MYSQL_DATABASE,
+    });
+
+    con.connect(function (err) {
+      if (err) throw err;
+    });
+    return con;
+  },
+
+  setupDB: async (con) => {
+    await con.promise().query({
+      sql: "CREATE TABLE IF NOT EXISTS profiles(name varchar(20) PRIMARY KEY, pings INT)",
+      timeout: 30000, // 30s
+    });
+  },
```

Comment:
nitpick: if we're only `async` to return a single function call marked with `await`, we can shorthand it a bit.  our goal is to return a promise (which happens automatically by marking the function as `async`).

```suggestion
  setupDB: (con) => con.promise().query({
    sql: "CREATE TABLE IF NOT EXISTS profiles(name varchar(20) PRIMARY KEY, pings INT)",
    timeout: 30000, // 30s
  }),
```